cypress/integration/functional/wallet/transaction/list.spec.ts

```diff
@@ -1,10 +1,29 @@
 context('Wallet - Transaction - List', () => {
   describe('wallet has 2 transactions in history', function () {
     before(() => {
+      cy.visit('/')
```

Comment:
@kyleleow this one is already done inside createEmptyWallet

Reply:
updated